package.json

```diff
@@ -2,11 +2,10 @@
   "name": "mdb-world-22-dashboard-embedding-tutorial",
   "version": "1.0.0",
   "description": "Dashboard Embedding tutorial for MongoDB World 2022",
-  "main": "index.js",
-  "source": "src/index.html",
+  "default": "index.html",
   "scripts": {
-    "start": "parcel src/index.html --open",
-    "build": "parcel build",
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html",
```

Comment:
that's so weird, so the issue was that parcel doesn't like `index.html` alongside the src directory?